src/languages/typescript.js

```diff
@@ -222,6 +222,9 @@ export default function(hljs) {
       {
         begin: '\\.' + hljs.IDENT_RE, relevance: 0 // hack: prevents detection of keywords after dots
       },
+      {
+        begin: '\\b[A-Za-z$_][0-9A-Za-z$_]*:', relevance: 0 // prevents detection of keywords in property name
```

Comment:
Please show an example or two of exactly what you’re trying to avoid by adding this.

Reply:
Pretty straightforward example:
`public readonly type: number;`

Reply:
I’m thinking this should probably reuse `ECMAScript.IDENT_RE`, isn’t that what that is?... you can use `regex.join` to glue the pieces together.

Reply:
Replaced it with simple concatenation